app/commands/codemod/review.go

```diff
@@ -61,12 +64,51 @@ func RequestReview(ctx context.Context, req *flow.ReviewRequest) (chan *flow.Iss
 		return nil, nil, errors.Trace(err)
 	}
 
-	issuec, errorc, err := c.Codemod(ctx, req)
+	payload, err := ptypes.MarshalAny(req)
+	if err != nil {
+		return nil, nil, errors.Trace(err)
+	}
+
+	// TODO(waigani) refactor review, search and codemod code out of client completely.
+	replyc, runErrc, err := c.Run(ctx, &flow.Request{Flow: "codemod", Payload: payload})
 	if err != nil {
 		return nil, nil, errors.Trace(err)
 	}
 
-	return issuec, errorc, nil
+	issuec := make(chan *flow.Issue)
+	errc := make(chan error)
+	wg := sync.WaitGroup{}
+	wg.Add(2)
+
+	go func() {
+		for err := range runErrc {
+			errc <- err
+		}
+		wg.Done()
+	}()
+
+	go func() {
+		for reply := range replyc {
+
+			var issue *flow.Issue
+			ptypes.UnmarshalAny(reply.Payload, issue)
+			if err != nil {
+				errc <- err
+				continue
+			}
+
+			issuec <- issue
+		}
+		wg.Done()
+	}()
+
+	go func() {
```

Comment:
Could replace the 3 go routines and wait group with a single routine and a for select loop.